### PR TITLE
Fix/quorum

### PR DESCRIPTION
**Pull request by dovgopoly**

### CODE REVIEW

File: contracts/libs/utils/TypeHelper.sol [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "../../interfaces/gov/IGovPool.sol";
+
+library TypeHelper {
```
Comment (Arvolear) [inline contracts/libs/utils/TypeHelper.sol:6]: Can `GovUserKeeperView` be optimized with this?

File: contracts/libs/gov/gov-pool/GovPoolCreate.sol [hunk 2]
```diff
@@ -154,20 +168,45 @@ library GovPoolCreate {
...
+        IGovPool govPool = IGovPool(address(this));
+
+        (, address userKeeperAddress, , , ) = govPool.getHelperContracts();
+        (, address expertNft, address dexeExpertNft, ) = govPool.getNftContracts();
+
+        IGovUserKeeper userKeeper = IGovUserKeeper(userKeeperAddress);
+
         for (uint256 i; i < actions.length; i++) {
             IGovPool.ProposalAction calldata action = actions[i];
+            bytes4 selector = action.data.getSelector();
```
Comment (Arvolear) [inline contracts/libs/gov/gov-pool/GovPoolCreate.sol:185]: Will this work with empty calldata? Like when ether is transferred

Comment (dovgopoly) [reply contracts/libs/gov/gov-pool/GovPoolCreate.sol:185]: It'll return bytes4(0) selector

File: contracts/libs/utils/TypeHelper.sol (hunk 1 above)
Comment (dovgopoly) [reply contracts/libs/utils/TypeHelper.sol:6]: done

**Pull request merged**